src/lib/y2storage/filesystems/legacy_nfs.rb

```diff
@@ -0,0 +1,230 @@
+# encoding: utf-8
+
+# Copyright (c) [2018] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2storage/filesystems/nfs"
+
+module Y2Storage
+  module Filesystems
+    # Class to represent an NFS mount, as defined in the old y2-storage.
```

Comment:
*a NFS mount*

Reply:
Sure? I had a similar discussion not long ago about "an LVM volume" vs "a LVM volume" and turned out the former was decided to be the right one. So I applied the same (lack of) rule here. Is supposed to be based on how the "average speaker" (if such thing exists) pronounces it.